action.yml

```diff
@@ -13,6 +13,14 @@ inputs:
     description: 'Kapeta credentials, typically a JWT token'
     required: true
 
+  staging:
```

Comment:
What's the staging property used for ? Isn't it "just" base_url that's needed?

Reply:
No, not quite. Base url is https://username:password@app.staging.kapeta.com - staging instructs the action to also write a registry.json to ~/.kapeta containing npm.staging.kapeta.com, etc.